VocaDbModel/Utils/AppConfig.cs

```diff
@@ -239,5 +239,9 @@ public static SongType[] SongTypes
 		public static string YoutubeApiKey => Val("YoutubeApiKey");
 
 		public static string VimeoApiKey => Val("VimeoApiKey");
+
+		public static string DiscordWebhookUrl = Val("DiscordWebhookUrl");

```

Comment:
Might be better to use "=>" here, same as others, so we don't need to read this for tests.

Reply:
And maybe add some comments here explaining the purpose

Reply:
Oh, thanks. This is due to my new keyboard.